packages/gatsby-remark-graphviz/src/index.js

```diff
@@ -10,25 +11,39 @@ module.exports = async ({ markdownAST }, pluginOptions = {}) => {
   let codeNodes = []
 
   visit(markdownAST, `code`, node => {
+    const chunks = (node.lang || ``).match(
+      /^(?<lang>\S+)(\s+(?<attrString>.+))?/
+    )
     // Only act on languages supported by graphviz
-    if (validLanguages.includes(node.lang)) {
-      codeNodes.push(node)
+    if (chunks && validLanguages.includes(chunks.groups.lang)) {
+      node.lang = chunks.groups.lang
+      codeNodes.push({ node, attrString: chunks.groups.attrString })
     }
     return node
   })
 
   await Promise.all(
-    codeNodes.map(async node => {
+    codeNodes.map(async ({ node, attrString }) => {
       const { value, lang } = node
 
       try {
         // Perform actual render
         const svgString = await viz.renderString(value, { engine: lang })
 
+        // Add default inline styling
+        const $ = cheerio.load(svgString)
+        $(`svg`).attr(`style`, `max-width: 100%; height: auto;`)
```

Comment:
it's probably a good way to introduce this feature in https://github.com/mdaines/viz.js itself, enabling it to allow custom attributes.